### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by ishimwepati**
Hi, 

Below are the features I implemented in this branch :

- I added routes for users
- I created an HTML structure for users
- I created post against user

Your suggestions are most welcome !

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#   # layout 'boilerplate'
+#   def index
+#     @users = User.order(id: :asc)
+#   end
+
+#   def show
+#     @users = User.find(params[:id])
+#   end
+# end
+
```
Comment (omar-muhamad) [inline app/controllers/users_controller.rb]: - [x] It's considered good practice to have a readable code that is clean from any commented(unused) code. Please remove this commented code.

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class PostsController < ApplicationController
+  # load_and_authorize_resourcecls
+
```
Comment (omar-muhamad) [inline app/controllers/posts_controller.rb]: - [x] Same here, It's considered good practice to have a readable code that is clean from any commented(unused) code. Please remove this commented code.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @ishimwepati,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (ishimwepati) [reply app/controllers/users_controller.rb]: Thanks

File: app/controllers/posts_controller.rb (hunk 2 above)
Comment (ishimwepati) [reply app/controllers/posts_controller.rb]: Done

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @ishimwepati,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**